api/apiServer.py

```diff
@@ -62,34 +62,36 @@ def root(ins_name: str):
     ec2 = boto3.resource('ec2')
     ec2.instances.filter(Filters=[{
        "Name": "tag:Name",
-       "Values": [ins_name]}]).terminate()
+       "Values": [ins_name]}]).terminate() # For terminating an specfic ec2 instance
     return  {"message": "{} AWS Instance Destroyed.".format(ins_name)}
     
-@app.get("/control/aws/startInstance")
-def root():
-    vagrantfile = "api/config/aws"
-    v = vagrant.Vagrant(vagrantfile, quiet_stdout=False)
-    v.up() 
-    return  {"message": "AWS Instance Started"}
+@app.get("/control/aws/startSpecficInstance")
```

Comment:
There is typo here. But, anyway let's change the status naming convention.
"specific instance" just "Instance"
"All the instance" use "InstanceAll"

Reply:
```suggestion
@app.get("/control/aws/startInstance")
```